app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -15,4 +18,16 @@ class VerifyCsrfToken extends Middleware
         // entry point...
         '/php/index.php'
     ];
+
+    public function handle(Request $request, Closure $next)
+    {
+        if ($request->is('api/*')) {
+            $apiKey = Configs::get_value('api_key');
+            if ($apiKey && $request->header('Authorization') === $apiKey) {
```

Comment:
Do you want it as an part of the header or would it be more interesting to have it as a `POST` field ? :)

Reply:
I'm not asking it to be changed, I am just genuinely curious. :smiley:

Reply:
I'm OK with either. I know the `Authorization` header is common, but that's also usually for JWTs or OAuth tokens, so having it as a header could be confusing to some people? I'm OK with whichever you prefer as it's an easy change.